src/main/java/com/google/api/gax/grpc/InstantiatingChannelProvider.java

```diff
@@ -191,9 +191,15 @@ static String getGaxVersion() {
   }
 
   private static String getGrpcVersion() {
-    String grpcVersion = ManagedChannel.class.getPackage().getImplementationVersion();
-    if (grpcVersion == null) {
-      grpcVersion = DEFAULT_VERSION;
+    String grpcVersion = DEFAULT_VERSION;
+    Properties gaxProperties = new Properties();
+    try {
+      gaxProperties.load(
+          InstantiatingChannelProvider.class
+              .getResourceAsStream("/com/google/api/gax/gax.properties"));
+      grpcVersion = gaxProperties.getProperty("grpc_version");
+    } catch (IOException e) {
+      e.printStackTrace(System.err);
```

Comment:
If there is an IO exception here, which means the properties may not be located, it would be useful to print the stack trace for debugging so I would like to keep the function here.